config/routes.rb

```diff
@@ -1,7 +1,7 @@
-Basechurch::Engine.routes.draw do
+Rails.application.routes.draw do
   resources :api_keys, except: [:new, :edit]
   devise_for :users,
-             class_name: 'Basechurch::User',
+             class_name: 'User',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.